unocss.config.ts

```diff
@@ -11,4 +19,28 @@ export default defineConfig({
     // See: https://unocss.dev/presets/web-fonts
     presetWebFonts(),
   ],
+  theme: {
+    colors: {
```

Comment:
Let's add here also the button color/weight/size as well as for the event badges colors, e.g. $nord14 => events, $nord13 => conference, $nord12 => hackathon and so on

Note that the $nord variables are just for examples here

Reply:
Imma search some colors and select some of them for specific types of events!